app/models/enforcement.rb

```diff
@@ -5,6 +5,9 @@ class Enforcement < ApplicationRecord
 
   include EnforcementStatus
 
+  has_one :case_record, as: :caseable
```

Comment:
`include Caseable` will already add the `has_one` so this line shouldn't be needed, I think.

```suggestion
```